pkg/crd/crd.go

```diff
@@ -68,7 +68,7 @@ func ApplyCRD(ctx context.Context, client extv1client.ApiextensionsV1beta1Interf
 }
 
 // WaitForCRDReady blocks until the CRD is ready.
-func WaitForCRDReady(ctx context.Context, client extv1client.ApiextensionsV1beta1Interface, crdName string) error {
+func WaitForCRDReady(ctx context.Context, client extv1client.ApiextensionsV1beta1Interface, crdName string) error { // nolint:interfacer
```

Comment:
Does anyone  intend to use the output of interfacer? Maybe we drop that linter

Reply:
@manno I had some feeling. 
BTW, interface is out of maintain: https://github.com/mvdan/interfacer
> Deprecated: A tool that suggests interfaces is prone to bad suggestions, so its usefulness in real code is limited.

Reply:
Ok, added a commit to drop it